### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by eternal-loading-screen**
Adding in assignment 1 answers

## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
answer the assignment

## What did you learn from the changes you have made?
The concept of creating a model to predict y based on X values, to fit the data, then run the results based on the test data inputs.

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
N/A

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
going from using train_test_split() to then using the fitted model that was based on our training data, and then running it on our test data inputs

## How were these changes tested?
running them via VS Code

## Checklist
- [ ] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by fyellow — CHANGES_REQUESTED** [review]
While the code is correct, there are a few conceptual mistakes. 
“Class field does not have any bearing on our analysis and should not be a meaningful consideration...”
This is incorrect - the class is the target we’re predicting, so it is extremely meaningful.

Why set seed? "Prevents human judgement" : Setting a random seed ensures reproducibility.

**Comment by eternal-loading-screen** [comment]
Hi @fyellow ,

Just following up on this, would you like me to update the answers on the assignment?

Also for Class field, it was initially difficult to put into context, what expressing the class field as a numeric value meant. (EX: is 0 better than 2)
